packages/web/src/common/store/account/selectors.ts

```diff
@@ -29,9 +30,11 @@ export const getAccountToCache = (state: CommonState) => ({
   hasFavoritedItem: state.account.hasFavoritedItem
 })
 
-export const getAccountUser = createSelector(
+export const getAccountUser = createDeepEqualSelector(
```

Comment:
Is this needed because the user object reference changes?

Reply:
i think youre right on this one, can double check